source/dem/velocity_verlet_integrator.cc

```diff
@@ -63,45 +60,53 @@ VelocityVerletIntegrator<dim>::integrate(
   std::vector<Tensor<1, 3>> &      force,
   const std::vector<double> &      MOI)
 {
+  Point<3>     particle_position;
+  Tensor<1, 3> dt_g = g * dt;
+
   for (auto &particle : particle_handler)
     {
       // Get the total array view to the particle properties once to improve
       // efficiency
-#if (DEAL_II_VERSION_MAJOR < 10 && DEAL_II_VERSION_MINOR < 4)
-      types::particle_index particle_id = particle.get_id();
-#else
       types::particle_index particle_id = particle.get_local_index();
-#endif
+
       auto          particle_properties = particle.get_properties();
       Tensor<1, 3> &particle_torque     = torque[particle_id];
       Tensor<1, 3> &particle_force      = force[particle_id];
-      Tensor<1, 3>  particle_acceleration;
-      Point<3>      particle_position;
-      double mass_inverse = 1 / particle_properties[PropertiesIndex::mass];
-      double MOI_inverse  = 1 / MOI[particle_id];
+
+      double dt_mass_inverse = dt / particle_properties[PropertiesIndex::mass];
+      double dt_MOI_inverse  = dt / MOI[particle_id];
 
       if constexpr (dim == 3)
         particle_position = particle.get_location();
 
       if constexpr (dim == 2)
         particle_position = point_nd_to_3d(particle.get_location());
```

Comment:
No lambda cool stuff here? :(

Reply:
and at the end of the code

Reply:
Not sure if the cool lambda stuff is more readable, but i'll add it since you seem to fancy that :P!

Reply:
The one at the end of the code is not an assignment so I am not sure the lambda is the best approach here :).